resources/zib/zib-AdministrationAgreement.xml

```diff
@@ -0,0 +1,274 @@
+    <element id="MedicationDispense.extension:periodOfUseDuration.value[x]">
+      <path value="MedicationDispense.extension.value[x]" />
+      <short value="PeriodOfUse / Duration" />
+      <definition value="**Duration**: The intended duration of use. E.g. 5 days or 8 weeks. It is not allowed to indicate the duration in months, because different months have a variable duration in days." />
+      <alias value="Gebruiksperiode" />
+      <mapping>
+        <identity value="zib-administrationagreement-v1.0.3-2020EN" />
+        <map value="NL-CM:9.8.22660" />
+        <comment value="PeriodOfUse" />
+      </mapping>
+    </element>
+    <element id="MedicationDispense.modifierExtension">
+      <path value="MedicationDispense.modifierExtension" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="MedicationDispense.modifierExtension:stopType">
+      <path value="MedicationDispense.modifierExtension" />
+      <sliceName value="stopType" />
+      <max value="1" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-StopType" />
+      </type>
+    </element>
+    <element id="MedicationDispense.modifierExtension:stopType.value[x]">
+      <path value="MedicationDispense.modifierExtension.value[x]" />
+      <short value="AdministrationAgreementStopType" />
+      <definition value="Stop type, the manner in which this medication is discontinued (temporary or definitive)." />
+      <alias value="ToedieningsafspraakStopType" />
+      <mapping>
+        <identity value="zib-administrationagreement-v1.0.3-2020EN" />
+        <map value="NL-CM:9.8.22498" />
+        <comment value="AdministrationAgreementStopType" />
+      </mapping>
+    </element>
+    <element id="MedicationDispense.modifierExtension:repeatPeriodCyclicalSchedule">
+      <path value="MedicationDispense.modifierExtension" />
+      <sliceName value="repeatPeriodCyclicalSchedule" />
+      <max value="1" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-InstructionsForUse.RepeatPeriodCyclicalSchedule" />
+      </type>
+    </element>
+    <element id="MedicationDispense.status">
+      <path value="MedicationDispense.status" />
+      <short value="CanceledIndicator" />
+      <definition value="A code specifying the state of the set of dispense events. In the event of an error correction, the value of this element shall be 'entered-in-error'." />
+      <alias value="GeannuleerdIndicator" />
+      <mapping>
+        <identity value="zib-administrationagreement-v1.0.3-2020EN" />
+        <map value="NL-CM:9.8.23034" />
+        <comment value="CanceledIndicator (if status = 'entered-in-error' then CanceledIndicator = true)" />
+      </mapping>
+    </element>

```

Comment:
The definition is not right the zib says:

"In the event of an error correction, this indicator is to be put on for the incorrect agreement"

Also there is still a ongoing discussion about the cardinality 

https://github.com/Nictiz/Nictiz-R4-zib2020/issues/97

Reply:
This is the only place where we put mapping instructions in .mapping.comment. I think the instructions should be put in .definition.

Re: Ashwin, I don't think further discussion on cardinality is needed. The system just needs to fill .status to the best of its capabilities.

Reply:
> I think the instructions should be put in .definition.
Or on .comment. Not sure actually

Reply:
Moved from the `mapping.comment` to comment on `status`.